RNASeqTools_TSS/rnaseqtools_tss.jl

```diff
@@ -0,0 +1,105 @@
+# # RNASeq data analysis using RNASeqTools
+#
+# Requirements:
+#
+# * Julia 1.8.5
+# * RNASeqTools 0.3.4
+# * sra-tools 3.0.3
+# * samtools 1.6
+# * bwa-mem2 2.2.1
+#
+# In this tutorial, we will first download a few files (genome, annotation, sequencing reads) with sra-tools. 
+# We will then align the reads to the genome using bwa-mem2. We will then use the various Packages from BioJulia 
+# through the interface of RNASeqTools to load the alignments file, compute the coverage from it and do some 
+# simple signal detection in the coverage.
+
+using Pkg
+Pkg.add(url="https://github.com/maltesie/RNASeqTools#v0.3.4")
+
+using RNASeqTools
+
+# ## Downloading the data
+#
+# Our data will be from V. Cholerae, we need a genome and its annotation: First go to [NCBI](https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_013085075.1/),
+# after clicking Download, select .fasta and .gff and click Download again. Then extract the content of the archive 
+# directory /ncbi_dataset/data/GCF_013085075.1/ into the folder containing this notebook. The folder should then
+# look like:
+#
+# RNASeqTools_TSS
+#   ├── GCF_013085075.1_ASM1308507v1_genomic.fna
+#   ├── genomic.gff
+#   ├── Manifest.toml
+#   ├── Project.toml
+#   ├── rnaseqtools_tss.ipynb
+#   └── rnaseqtools_tss.jl
+
+# NCBI stores reads in its internal format SRA, we can use prefetch and fastq-dump from the sra-tools to 
+# download reads and convert them to the fastq format. The reads in sample SRR1602510 come from a TEX-treated 
+# sample and can be used to identify primary transcription start sites.
+
+download_prefetch(["SRR1602510"], @__DIR__)
```

Comment:
Is this defined in your package? I think `BioServices.jl` has some related functionality.

Reply:
The function uses prefetch to download the sra internal format and converts this to a fastq file using fastq-dump. I had another look into BioServices.jl and how I understand it, the EUtils module can be used to search SRA and also retreive links to the files, but I could not find anything in there that would allow for a conversion to a readable format. Did I miss it?